packages/sentry/package.json

```diff
@@ -7,12 +7,12 @@
    "dependencies": {
       "@sentry/nextjs": "^7.0.0"
    },
-   "peerDependencies": {
-      "next": "11.1.3",
-      "react": "17.0.2"
-   },
    "devDependencies": {
+      "@babel/core": "^7.0",
       "@ifixit/tsconfig": "workspace:*",
+      "react": "^17.0",
+      "react-dom": "^17.0",
+      "next": "^11.0",
```

Comment:
For context, `react` and `next` are peer dependencies of `@sentry/nextjs`, and `react-dom` and `@babel/core` are peer dependencies of `react`/`next`.